File: python/1071.py
x = int(input())
y = int(input())

sum = 0

if x > y:
    greater = x
    lower = y
else:
    greater = y
    lower = x

for i in range(lower + 1, greater):
    if i % 2 == 1:
        sum += i
        
print(sum)

File: lua/1071.lua
local x = io.read('n')
local y = io.read('n')

local sm
local lg

if x > y then
    sm = y
    lg = x
else
    sm = x
    lg = y
end

local sum = 0
local i = sm + 1
while i < lg do
    if i % 2 == 1 then
        sum = sum + i
    end
    i = i + 1
end
print(sum)